TicTacToe.Tests/BoardTests.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using Xunit;
+
+namespace TicTacToe.Tests
+{
+    public class BoardTests
+    {
+        BoardFactory boardFactory = new BoardFactory();
+
+        [Fact]
+        public void CreateNew3x3Board()
+        {
+            // BoardFactory boardFactory = new BoardFactory();
```

Comment:
Is there a reason these are commented out?

Reply:
Like the test below, I commented them out and simply forgot to remove them - I'll try to be more consistent in removing those comments before I commit!

Reply:
@digitopolis cool, sounds good!